### PR TITLE
Correct pricings

### PR DESCRIPTION
**Pull request by NotJansel**
Mini did a slight mathmematical mistake, I corrected it with this pr

### CODE REVIEW

File: docs/minipad/purchase-pricing.md [hunk 1]
```diff
@@ -50,8 +50,8 @@ Here's an overview of the estimated pricing on the v1 version of the minipad fea
 | :-------:| :--------: | :---------: |
 | 5x Case | 26.15€ | 5.23€ |
 | 5x PCB | 36.96€ | 7.39€ |
-| 24x Switches | 18.98€ | 3.80€ |
-| **Total**| **82.09€** | **16.42€** per person |
+| 24x Switches | 19.98€ | 4€ |
```
Comment (minisbett) [inline docs/minipad/purchase-pricing.md]: Can you put `4.00€`?

**Review by minisbett — CHANGES_REQUESTED** [review]

**Comment by minisbett** [comment]
Thanks for PRing this!

**Comment by NotJansel** [comment]
like this?

**Comment by minisbett** [comment]
yesh, thanks!

**Pull request merged**